extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/AtlasETC1CompressorTest.java

```diff
@@ -0,0 +1,135 @@
+
+package com.badlogic.gdx.tools.etc1;
+
+import java.io.File;
+
+import com.badlogic.gdx.ApplicationListener;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Pixmap.Format;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.math.Matrix4;
+import com.badlogic.gdx.tools.texturepacker.TexturePacker;
+import com.badlogic.gdx.tools.texturepacker.TexturePacker.Page;
+import com.badlogic.gdx.utils.Array;
+
+public class AtlasETC1CompressorTest implements ApplicationListener {
+
+	private String atlas_path;
+	private FileHandle atlas_file;
+
+	public AtlasETC1CompressorTest (String atlas_path) {
+		this.atlas_path = atlas_path;
+		this.atlas_file = new FileHandle(atlas_path);
+	}
+
+	public static void main (String[] args) throws Exception {
+
+		String application_home_path_string = System.getProperty("user.dir");

```

Comment:
It's not user's home directory. This is a complete absolute path from where the application was initialized.